tests/structuredheadings/formatcombo-boundaries.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable */
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: wysiwygarea,structuredheadings,toolbar,basicstyles,dialog,richcombo,undo */
+/* eslint-enable */
+
+// Clean up all instances been created on the page.
+(function () {
+  var comboName = "NumFormats";
+
+  bender.editor = {
+    allowedForTests: "h1; h2; p"
+  };
+
+  bender.test({
+    setUp: function () {
+      //Anything to be run before each test if needed
+    },
+
+    tearDown: function () {
+      var editor = this.editorBot.editor;
+      editor.plugins.structuredheadings.currentScheme = "1.1.1.1.1.";
```

Comment:
It's odd to me that the tearDown is responsible for setting the scheme (versus the setUp). Seems like that pattern makes it easy for test bleedthrough to occur implicitly.